gems/aws-sigv4/sig/request.rbs

```diff
@@ -0,0 +1,24 @@
+module AWS
+  module Sigv4
+    class Request
+      def initialize: (
+          http_method: String,
```

Comment:
This rbs syntax cannot support the following ruby syntaxes in steep.

```rb
# With Hash argument
hash = {http_method: 'get'}
Aws::Sigv4::Request.new(hash)

# With Hash directly
Aws::Sigv4::Request.new({http_method: 'get'})
```

I solve this problem this way.

```rb
def initialize: (?foo: String, ?bar: Integer, ?baz: bool) -> void # Support keyword arguments
              | (?Hash[Symbol, untyped]) -> void                  # Support others
```

This is true for almost all methods in aws-sdk.

Reply:
Fixed!
a522cc1b8a06496539730bbfedd6ab3110050792